Servers/UI/OJS.Servers.Ui/ClientApp/src/common/register-contest-helper.ts

```diff
@@ -0,0 +1,7 @@
+import { ContestParticipationType } from './constants';
+
+export const canCompeteContest =
```

Comment:
Rename file to contest helpers. Also then naming of the function is wrong according to what it does. Compete is participationType === ContestParticipationType.Compete. Looking at the way you are using this function, it looks more like isParticipationTypeValid